modules/ROOT/pages/to-request-access.adoc

```diff
@@ -4,13 +4,25 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-APIs that have been protected by a Client ID enforcement policy require API consumers to create an application to hold the Client ID and Secret. This lets the app access the API as well as provide consumer analytics for an API developer.
+When you request access, the client application is registered and creates an
+API contract between the API and the client.
```

Comment:
creates a contract between the API and client application--not an API contract.

Reply:
Fixed.